components/sup/src/ctl_gateway/mod.rs

```diff
@@ -22,37 +22,32 @@
 
 pub mod server;
 
-use std::{borrow::Cow,
-          fmt,
+use std::{fmt,
           fs::{self,
                File},
           io::{self,
                Write},
           path::Path};
 
-use regex::Regex;
-use termcolor::{Buffer,
-                Color,
+use termcolor::{Color,
+                ColorChoice,
                 ColorSpec,
+                StandardStream,
                 WriteColor};
 
 use crate::{api_client::DisplayProgress,
-            common::ui::UIWriter,
-            hcore::{self,
-                    output},
+            common::{output::{self,
+                              OutputFormat,
+                              StructuredOutput},
+                     ui::UIWriter,
+                     PROGRAM_NAME},
+            hcore,
             protocol};
 use futures::prelude::*;
 
 use crate::error::{Error,
                    Result};
 
-lazy_static! {
-    /// Shamelessly stolen from https://github.com/chalk/ansi-regex/blob/master/index.js
-    static ref STRIP_ANSI_CODES: Regex = Regex::new(
-        r"[\x1b\x9b][\[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-PRZcf-nqry=><]")
-        .unwrap();
```

Comment:
We don't need this anymore?

Reply:
nope. because now we don't have a scenario where we have injected ansi strings and later need to remove them. Using `termcolor` they are injected at the time the output is streamed based on the color preferences of the environment.

Reply:
That's fun! 😄